styles/Credits.module.css

```diff
@@ -0,0 +1,24 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  overflow-y: auto;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+.container::-webkit-scrollbar {
+  display: none;
+}
+
+.title {
+  color: var(--secondary-color);
+  font-family: Poppins, sans-serif;
```

Comment:
You could also create a font-family variable in the globals.css, if you use the same font-families often.

Reply:
Will do thank you!